src/index.html

```diff
@@ -1,13 +1,72 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="page__body">
+    <div class="card" data-qa="card">
+      <div class="card__im">
+        <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="imac"
+      >
+      </div>
+
+      <div class="product">
+        <h2 class="product__name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h2>
+
+        <p class="product__code">
+          Код товара: 195434
+        </p>
+
+        <div class="product__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="product__raiting">
+            Отзывов: 5
+          </p>
+        </div>
+
+        <div class="product__price">
+          <p class="price">
+            Цена:
+          </p>
+
+          <p class="value">
+            69 999 грн
+          </p>
+        </div>
+
+        <div class="product__button">
+          <a
+            class="button"
+            href="#"
+            data-qa="hover"
+          >
+            <p>купить</p>
```

Comment:
I think you could avoid wrapping it with `p`

Reply:
that's redundant `p` wrapper